github.go

```diff
@@ -16,6 +16,10 @@ type Repositories interface {
 	CreateStatus(owner, repo, ref string, status *github.RepoStatus) (*github.RepoStatus, *github.Response, error)
 }
 
+type Issues interface {
+	AddLabelsToIssue(owner, repo string, number int, labels []string) ([]github.Label, *github.Response, error)
```

Comment:
I think `number` is good enough here, as it's pretty much just documentation - no code uses it. And as I said elsewhere, I'd like to stay consistent with the go-github documentation.